lib/src/link/Link.tsx

```diff
@@ -78,6 +78,7 @@ const LinkContainer = styled.div<LinkContainerProps>`
     padding: 0;
     cursor: pointer;
     font-family: inherit;
+    font-size: inherit;
```

Comment:
Shouldn't this rule have the token value? Same question goes for font-family, font-style, font-weight, etc.

Reply:
Currently they are at `LinkText` but they won't affect every version of the link, no? 🤔

Reply:
Done! ✌🏼